plugins/azure_ad_admin/icon_azure_ad_admin/actions/create_user/action.py

```diff
@@ -47,10 +48,10 @@ def __init__(self):
     def run(self, params={}):
         passwd = _pw_gen()
 
-        user_created = self._create_user(params, passwd)
+        self._create_user(params, passwd)
 
-        message_to_send = self.compose_email(params, passwd)
-        result = self.send_message(message_to_send, params.get(Input.NOTIFY_FROM))
+        message_to_send = self._compose_email(params, passwd)
```

Comment:
Wow. How did this work before?? Good catch!

Reply:
It shouldn't works